WordPress/Classes/ViewRelated/Activity/RewindStatus+multiSite.swift

```diff
@@ -0,0 +1,16 @@
+import WordPressKit
+
+extension RewindStatus {
+    func isMultiSite() -> Bool {
+        reason == "multisite_not_supported"
+    }
+
+    func isActive() -> Bool {
+        state == .active
+    }
+
+    enum Strings {
+        static let multisiteNotAvailable = NSLocalizedString("Jetpack Backup for Multisite installations provides downloadable backups, no one-click restores. For more information visit our documentation page.", comment: "Message for Jetpack users that have multisite WP installation, thus Restore is not available.")
+        static let multisiteNotAvailableHighlight = NSLocalizedString("visit our documentation page", comment: "Portion of a message for Jetpack users that have multisite WP installation, thus Restore is not available. This part is a link, colored with a different color.")
```

Comment:
I wonder if it's safer to compose the the full string by creating a String format with a placeholder for the highlighted substring, like this.

https://github.com/wordpress-mobile/WordPress-iOS/blob/11ea6606810bbea6bbc1beff1002d03947dc861a/WordPress/Classes/ViewRelated/Jetpack/Jetpack%20Scan/View%20Models/JetpackScanThreatViewModel.swift#L286-L287

My concern is that if the translation for the substring doesn't match up exactly between `multisiteNotAvailable` and `multisiteNotAvailableHighlight`, then the text won't be highlighted properly.

At the same time though, I don't think this is a huge issue because even if the substring doesn't get highlighted the button will still be tappable. Just something that caught my eye.

Reply:
Good point!

Done in fd73b0fa6d53ac6400c72093348314f4136191b0